plugins/chat/lib/chat/duplicate_message_validator.rb

```diff
@@ -2,45 +2,26 @@
 
 module Chat
   class DuplicateMessageValidator
-    attr_reader :chat_message
+    attr_reader :chat_message, :chat_channel
 
     def initialize(chat_message)
       @chat_message = chat_message
+      @chat_channel = chat_message&.chat_channel
     end
 
     def validate
-      return if SiteSetting.chat_duplicate_message_sensitivity.zero?
```

Comment:
should we remove this site setting? is it still used?

Reply:
I'd rather do that in another PR. First PR is to "deprecate" the behavior, second PR will be to "clean up" the DB.